concordia/static/js/action-app/index.js

```diff
@@ -792,6 +806,29 @@ export class ActionApp {
         return sortBy(assetList, keyFromAsset);
     }
 
+    getSelectedOptionType() {
+        let campaignOrTopic = 'campaign';
+
+        if (
+            this.campaignSelect.options[
+                this.campaignSelect.selectedIndex
+            ].parentElement.classList.contains('topic-optgroup')
```

Comment:
I'd probably favor a class name which doesn't describe the tag (e.g. `topics`) but I'm not sure it's that big a deal.